keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/CombinedRepository.kt

```diff
@@ -313,7 +316,10 @@ class CombinedRepository(
     resourceRepository.getResourcesByApplication(application)
 
   override fun getResourceSummaries(deliveryConfig: DeliveryConfig): List<ResourceSummary> =
-    resourceRepository.getResourceSummaries(deliveryConfig)
+    getResourcesByApplication(deliveryConfig.application)
```

Comment:
There was no value in adding an additional layer of indirection to get resource summaries from a `ResourceRepository`, plus I needed to use `ActuationPauser` below to determine status (which goes into the summaries), so that logic resides here now.